vantage6/cli/node.py

```diff
@@ -356,16 +358,26 @@ def cli_node_start(name, config, environment, system_folders, image, keep,
     }
 
     # only mount the DB if it is a file
-    info("Setting up database")
-    db_is_file = Path(ctx.databases["default"]).exists()
-    env['DATABASE_URI'] = "/mnt/database.csv" if db_is_file else \
-        ctx.databases['default']
-    info(f" - URI: {env['DATABASE_URI']}")
-    if db_is_file or skipp_db_exist_check:
-        mounts.append(("/mnt/database.csv", str(ctx.databases["default"])))
-    else:
-        warning(' - Are you using a non file-based database?')
-        warning('   Or did you make a mistake in the database path?')
+    info("Setting up databases")
+    db_labels = ctx.databases.keys()
+    for label in db_labels:
+
+        uri = ctx.databases[label]
+        info(f"  Processing database '{label}:{uri}'")
+        LABEL = label.upper()
+
+        file_based = Path(uri).exists()
+        if not file_based and not force_db_mount:
+            debug('  - non file-based database added')
+            env[f'{LABEL}_DATABASE_URI'] = uri
+        else:
+            debug('  - file-baased database added')
+            env[f'{LABEL}_DATABASE_URI'] = f'/mnt/{label}.csv'
+            mounts.append((f'/mnt/{label}.csv', str(uri)))
+
+        # FIXME legacy to support < 2.1.3 can be removed from 3+
+        if label == 'default':
+            env['DATABASE_URI'] = '/mnt/default.csv'
```

Comment:
Shouldn't this be '/mnt/database.csv'?
And this database is now not mounted under the same path as the original, should we mount it twice?

Reply:
No, since the label is `default` the URI needs to be the same. Previous version read this `DATABASE_URI` and will mount it in the algorithm container.